inbm/dispatcher-agent/dispatcher/sota/os_factory.py

```diff
@@ -247,5 +250,39 @@ def create_snapshotter(self, sota_cmd: str, snap_num: Optional[str],
         return WindowsSnapshot(trtl, sota_cmd, snap_num,
                                proceed_without_rollback, reboot_device)
 
-    def create_downloader(self) -> Downloader:
+    def create_downloader(self, signature: Optional[str] = None) -> Downloader:
```

Comment:
Document that signature is ignored in doc string.

Reply:
Added